### ISSUE TITLE
cannot find child element XamlFilename of element Workflow

### ISSUE THREAD

**Issue by udayveluvali** [scottdurow/SparkleXrm#350]
I am trying to export a solution which contains flows and I am getting below error.

![image](https://user-images.githubusercontent.com/30835014/70769697-9295ac00-1d90-11ea-8843-85b6f4b043f2.png)

**Comment by namurdock** [comment]
Are you on the latest version of coretools from Nuget? That is usually the source of the problem.

**Comment by udayveluvali** [comment]
Yes, I have installed the latest core tools and latest stable version as well.

**Comment by dangerousromeo** [comment]
Same problem..Any update

**Comment by dangerousromeo** [comment]
> Yes, I have installed the latest core tools and latest stable version as well.

Try installing through visual studio package manager...it worked for me
